tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -413,6 +413,18 @@ _ when x.ToBoolean() is { } boolean => boolean,
                                    .AsBoolResult()
                                    .OverrideWith(in otelRuntimeMetricsEnabled, ErrorLog, defaultValue: false);
 
+            DataPipelineEnabled = config
+                                  .WithKeys(ConfigurationKeys.TraceDataPipelineEnabled)
+                                  .AsBool(defaultValue: true);
+
+            // Due to missing quantization and obfuscation in native side, we can't enable the native trace exporter
+            // as it may lead to different stats results than the managed one.
+            if (DataPipelineEnabled && StatsComputationEnabled)
+            {
+                DataPipelineEnabled = false;
+                _telemetry.Record(ConfigurationKeys.TraceDataPipelineEnabled, false, ConfigurationOrigins.Calculated);
```

Comment:
Should we log this?

Reply:
Similar question if it is enabled but we fallback for other reasons (e.g. native library not found).